src/screens/Home.tsx

```diff
@@ -1,66 +1,72 @@
-import { StatusBar } from 'expo-status-bar';
+import { StatusBar } from "expo-status-bar";
 import React, { useState } from "react";
-import { StyleSheet, Text, View, Pressable } from 'react-native';
-import { useNavigation } from '@react-navigation/native';
-import SearchBar from '../components/SearchBar';
+import { StyleSheet, Text, View, Pressable } from "react-native";
+import { useNavigation } from "@react-navigation/native";
+import SearchBar from "../components/SearchBar";
+import { getLocationsAsync, useAppDispatch } from "@store";
 
 type Nav = {
   navigate: (value: string) => void;
-}
-  
+};
+
 export default function Home() {
+  const [searchText, setSearchText] = useState("");
```

Comment:
Shouldn’t the location state also live in the redux store?

Reply:
The location is in the store. 🤔Are we talking about different things here?

Reply:
Yes, you're right, my mistake. We'll need to refactor the search query hook to use the redux store in another ticket.